manager/src/main/java/com/redhat/service/bridge/manager/ShardService.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.service.bridge.manager;
+
+public interface ShardService {
+
+    String getAssignedShardId(String id);
```

Comment:
Both at the moment. Since the assignment logic still needs to be implemented I'd say the signature of the method will change for sure (for example it will take into account also the `customerId`).